src/Parser/Lexer/Package.idr

```diff
@@ -0,0 +1,68 @@
+module Parser.Lexer.Package
+
+import public Parser.Lexer.Common
+import public Text.Lexer
+import public Text.Parser
+
+import Data.List
+import Data.Strings
+import Data.String.Extra
+import Utils.String
+
+%default total
+
+public export
+data Token
+  = Assignment
+  | Comment String
+  | EndOfInput
+  | DotSepIdent (List String)
+  | Ident String
+  | Separator
+  | Space
+  | StringLit String
+
+public export
+Show Token where
+  show Assignment = "Assignment"
+  show (Comment str) = "Comment: " ++ str
+  show EndOfInput = "EndOfInput"
+  show (DotSepIdent dsid) = "DotSepIdentifier: " ++ dotSep dsid
+  show (Ident i) = "Identifier: " ++ i
+  show Separator = "Separator"
+  show Space = "Space"
+  show (StringLit s) = "StringLit: " ++ s
+
+assignment : Lexer
+assignment = is '='
```

Comment:
This should probably be called `equals` or something like that; not all uses of an equals sign are assignments (and they are not in package descriptions).

Also, an assignment (statement) would normally consist of a LHS, an equals sign, and a RHS, such as `x = 3`; but here, we're just talking about the symbol itself.